types/react-native/test/index.tsx

```diff
@@ -447,6 +449,25 @@ class TextInputRefTest extends React.Component<{}, {username: string}> {
     }
 }
 
+class TextInputFocusBlurEventTest extends React.Component {
+    handleOnBlur(e: NativeSyntheticEvent<TextInputFocusEventData>) {
+    }
+
+    handleOnFocus(e: NativeSyntheticEvent<TextInputFocusEventData>) {
+    }
+
+    render() {
+        return (
+            <View>
+                <TextInput
+                    onBlur={(e: NativeSyntheticEvent<TextInputFocusEventData>) => this.handleOnBlur(e)}
+                    onFocus={(e: NativeSyntheticEvent<TextInputFocusEventData>) => this.handleOnFocus(e)}
```

Comment:
`e` didn’t need to be explicitly typed here, right?

This would have failed if it was typed with a different type, though, so it’s good in terms of testing 👍

Reply:
Yup, exactly.  It should be typed explicitly in the tests IMO